script/update-data.js

```diff
@@ -8,7 +8,7 @@ const packages = [
 ]
 
 const packagesString = packages.map((str) => `${str}@latest`).join(' ')
-const command = `yarn install ${packagesString} --exact`
+const command = `yarn add ${packagesString} --exact`
```

Comment:
This script needed to be fixed because usage of the `yarn install` command to add packages was removed at some point.